README.md

```diff
@@ -65,6 +65,7 @@ requirements:
 - Ruby: 2.4+
   - Bundler 2.1.4+
 - Rust: 1.46+
+- Go: 1.12+
```

Comment:
How did you/we determine this? Do we test on this version?

Reply:
This is the first version that go modules were officially released.